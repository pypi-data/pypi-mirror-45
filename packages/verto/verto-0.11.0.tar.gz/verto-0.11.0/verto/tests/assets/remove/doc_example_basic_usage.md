<div>
<remove>
<p>Content in here.</p>
{{ Django Variable }}
<p>Something about the variable.</p>
</remove>
</div>
