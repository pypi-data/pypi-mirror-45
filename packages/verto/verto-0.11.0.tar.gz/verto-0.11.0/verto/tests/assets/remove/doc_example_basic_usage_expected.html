<div>
<p>Content in here.</p>
{{ Django Variable }}
<p>Something about the variable.</p>
</div>
